infrastructure/networked_rds/bastion.tf

```diff
@@ -0,0 +1,138 @@
+data "aws_ami" "ami" {
```

Comment:
oh i think i see what's happening -- it's not necessary to copy and paste the source of the terraform-aws-postgres-vpc module -- that is accomplished in the `module {}` declaration. terraform will fetch those file from github or whatever `source = ` you put. I suggest you remove these files. If there are any specific pieces or extensions to the postgres vpc module, you could include them here or see if we can pull them into the module.

Reply:
I didn't understand this.